### ISSUE TITLE
Vmmem process keeps re-appearing without actually using it?

### ISSUE THREAD

**Issue by lesleyrs** [microsoft/WSL#9604]
### Version

Windows 10 - Version 10.0.19045.2486

### WSL Version

- [X] WSL 2
- [ ] WSL 1

### Kernel Version

5.15.79.1-microsoft-standard-WSL2

### Distro Version

Debian

### Repro Steps

Start pc, open task manager. Sometimes it takes a little to show up.

### Expected Behavior

I was expecting `Vmmem` process to not show up if not using WSL, or if executing `wsl --shutdown`?

### Actual Behavior

`Vmmem` process takes quite a bit of memory, sometimes disappears but also reappears without doing anything. Is this intentional or not? I should add that the process disappears when using `wsl --shutdown` but it comes back after a few seconds.

### Diagnostic Logs

`wsl -l -v` prints:

`* Debian    Stopped         2`

**Comment by OneBlue** [comment]
Thanks for reporting this @lesleyrs.

Can you share a /dumps when you see vmmem running ?

**Comment by lesleyrs** [comment]
@OneBlue I already found what the problem was and it's really stupid.

On google I saw somebody mention that if you run an application related to your distro it will restart, and I noticed when the process appears it does actually start running again.

Turns out that if you have a windows terminal profile (made it manually) for your distro that's enough for it to be running... I prefer running WSL from powershell so I deleted it.

I think that fixed it but I still saw the process appear for a little bit? But it doesn't stay around now so i'll close this.
